pkg/storage/couchdb/couchdbstore.go

```diff
@@ -355,6 +359,42 @@ func (c *CouchDBStore) getStoredValueFromRawDoc(rawDoc map[string]interface{}, k
 	return strippedJSON, nil
 }
 
+func (c *CouchDBStore) getRevision(k string) (string, error) {
+	rawDoc, err := c.getRawDoc(k)
+	if err != nil {
+		return "", err
```

Comment:
There are many unwrapped errors in this file

Reply:
Done - for all the changed code, I've made sure the errors are wrapped now. For unwrapped errors in the rest of the file: I will push a separate PR to fix this in order to keep this PR more focused.